component/card/card.css

```diff
@@ -0,0 +1,123 @@
+.card {
+    width: 30%; 
+    min-height: 40rem;
+    display: flex;
+    flex-direction: column;
```

Comment:
why assign a fixed width, you can go for min-width instead
and flex declaration can come from a utility instead of repeating here

Reply:
Yeah, will change that. This is the only place I used flex-direction: column. So having it in utility class did not make sense to me. Although, if I use this property some where else in the component library, I will change this.